src/api/giphy.js

```diff
@@ -0,0 +1,4 @@
+const HOST = "https://api.giphy.com/";
+const API_KEY = "Zx5PYfLyHZFhzV4bFwtohSoObgCckVUU";
```

Comment:
Use .env variables to save your api keys and do not expose them versioning them. You can provide an .env.example file expressing its usage.

https://www.npmjs.com/package/dotenv

Reply:
Modified, thanks!